doc/python.rst

```diff
@@ -74,7 +74,7 @@ results as `Numpy`_ arrays:
 
     json = """
     {
-      [
```

Comment:
I think the intent of the original change was to remove the base object and "pipeline" key.

Reply:
Right, removing the opening and closing {} then as {[]} is not valid JSON 5ec3f92